routes/definitions.js

```diff
@@ -79,6 +79,13 @@ router.post(
   })
 )
 
+router.post('/find', asyncMiddleware(findDefinitions))
```

Comment:
'find' feels not RESTful. Couple of thoughts

* could we do GET with query parameters?
* if POST is the way to go, can we hang this off `/` and basically have two different body formats, the array of coords, and this new one.

Reply:
makes sense. Let's do the GET with query parameters

Reply:
makes sense. Let's do the POST and check for array vs object for the query vs list.
The GET with query parameters is used to list coordinates from search